action.yml

```diff
@@ -129,6 +135,19 @@ runs:
         tutor local exec cms pip install -e /openedx/${{ inputs.app_name }}/
       shell: bash
 
+    - name: Install Inline Tutor plugins
+      if: ${{ inputs.inline_tutor_plugins_folder }}
+      run: |
+        source .tutor_venv/bin/activate
+
+        cp -r ${{ inputs.app_name }}/${{ inputs.inline_tutor_plugins_folder }}/* plugins/
+
+        for plugin_file in plugins/*; do
```

Comment:
Looks good! I think we just need to update the readme

Reply:
Done: https://github.com/eduNEXT/integration-test-in-tutor/commit/e0e2168a410ceff814622955dc628d6f1be0cf4a